a/afero/afero_ubi_8.5.sh

```diff
@@ -19,7 +19,7 @@
 # ------------------------------------------------------------------------------
 
 PACKAGE_NAME=github.com/spf13/afero
-PACKAGE_VERSION=${1:-v1.9.2}
+PACKAGE_VERSION=${1:-v1.6.0}
```

Comment:
let's not change the default version from high to low, let keep this at 1.9.2

Reply:
package version has been changes to 1.9.2